### PR TITLE
Ensure attachments always have a valid content type.

### PR DESCRIPTION
**Pull request by charlesmchen**
PTAL @michaelkirk

### CODE REVIEW

File: SignalServiceKit/src/Messages/Interactions/TSOutgoingMessage.m [hunk 1]
```diff
@@ -541,6 +541,8 @@ - (OWSSignalServiceProtosAttachmentPointer *)buildAttachmentProtoForAttachmentId
 
     OWSSignalServiceProtosAttachmentPointerBuilder *builder = [OWSSignalServiceProtosAttachmentPointerBuilder new];
     [builder setId:attachmentStream.serverId];
+    NSString *contentType = attachmentStream.contentType;
```
Comment (michaelkirk) [inline SignalServiceKit/src/Messages/Interactions/TSOutgoingMessage.m]: Did you want to do something with this var?

**Comment by michaelkirk** [comment]
Consider using `OWSMimeTypeApplicationOctetStream`

File: SignalServiceKit/src/Messages/Interactions/TSOutgoingMessage.m (hunk 1 above)
Comment (charlesmchen) [reply SignalServiceKit/src/Messages/Interactions/TSOutgoingMessage.m]: Removed.

**Comment by charlesmchen** [comment]
> Consider using OWSMimeTypeApplicationOctetStream

Done.

**Pull request merged**